test/config-luatest/vars_test.lua

```diff
@@ -137,6 +142,11 @@ g.test_sharding = helpers.run_as_script(function()
     local cluster_config = require('internal.config.cluster_config')
     local configdata_lib = require('internal.config.configdata')
     local t = require('myluatest')
+    local log = require('log')
+
+    -- Luatest logs assertions at info level. And run_as_script
+    -- asserts that stderr is empty.
+    log.cfg{level = 'error'}
```

Comment:
Can we adjust it in the helper itself?

For example, if luatest allows to disable extra logs using an environment variable, we can pass the env variable to the `justrun.tarantool` call in the `run_as_script` helper.

(Tarantool's logger itself follows env vars (#6011), but there are cases, when this feature doesn't work (#8092).)

Or we can perform the logger reconfiguration in the `myluatest` script and update tests that use `luatest` module directly.

What do you think?

(Feel free to discard if it requires a significant effort.)

Reply:
@ochaplashkin met the same problem and I guess that he is working on a solution on the luatest side. Oleg, please, confirm or correct me.

Reply:
Changed as discussed f2f. Test only exit code but print `stdout` and `stderr` in case of test failure.